lib/arpv4.ml

```diff
@@ -103,6 +103,8 @@ module Make (Ethif : V1_LWT.ETHIF) (Clock : V1.CLOCK) (Time : V1_LWT.TIME) = str
     | Not_found ->
       Hashtbl.replace t.cache ip (Confirmed (expire, mac))
 
+  let add t ip mac = notify t ip mac
```

Comment:
also, static arp entries shouldn't be overwritten by incoming packets (to prevent arp spoofing)